src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -121,6 +123,20 @@
             optional = true)
     protected int numReducers = 0;
 
+    @Argument(fullName = StandardArgumentDefinitions.CREATE_OUTPUT_BAM_INDEX_LONG_NAME,
+            shortName = StandardArgumentDefinitions.CREATE_OUTPUT_BAM_INDEX_SHORT_NAME,
+            doc = "If true, create a BAM index when writing a coordinate-sorted BAM file.", optional = true, common = true)
```

Comment:
This is a stylistic comment, but generally speaking I think the argument system in gatk makes the best option for boolean arguments that should be enabled by default a negative argument. As the user will have to type `--create-output-bam-splitting-index false` (etc for these other arguments) to the command line which is counterintuitive. I would probably change these to "disable-bam-splitting-index-creation" or some such if the default behavior is to create an index.